src/controllers/action-queue/cleaners.js

```diff
@@ -199,3 +199,17 @@ export const asActionProgramState: Cleaner<ActionProgramState> = asObject({
   programId: asString,
   effect: asOptional(asActionEffect)
 })
+
+// -----------------------------------------------------------------------------
+// Wyre Network Cleaners
+// -----------------------------------------------------------------------------
+
+export const asPaymentMethodsResponse = asObject({
+  data: asArray(
+    asObject({
+      id: asString,
+      // ...some other stuff
+      blockchains: asOptional(asObject(asString))
+    })
+  )
+})
```

Comment:
I feel like this goes in the WyreClient file, since it's an internal type used by that code. This cleaners file is more for common types used in multiple places.

Reply:
I moved this to the bottom of `WyreClient.js` file